src/server/server.cc

```diff
@@ -1839,6 +1844,26 @@ void Server::cleanupExitedWorkerThreads(bool force) {
   }
 }
 
+Status Server::overrideClusterMode() {
```

Comment:
Could you give it a better name? IMHO "override" is not appropriate here.

Reply:
How about checkClusterMode or validateClusterMode if it throws an error once the cluster mode is mismatched between the db and the configuration file?

Reply:
I think `checkClusterMode` would be better. Changed accordingly.